tests/test_modeling_utils.py

```diff
@@ -201,6 +203,7 @@ def forward(self, mask, inputs_embeds):
 
 TINY_T5 = "patrickvonplaten/t5-tiny-random"
 TINY_BERT_FOR_TOKEN_CLASSIFICATION = "hf-internal-testing/tiny-bert-for-token-classification"
+TINY_LLAMA = "trl-internal-testing/tiny-random-LlamaForCausalLM"
```

Comment:
```suggestion
TINY_MISTRAL = "hf-internal-testing/tiny-random-MistralForCausalLM"
```
Sorry let's use this one instead (the former one was from TRL library)